main.py

```diff
@@ -45,6 +45,9 @@ def main():
                     data_management.view_trans_by_date_range(df)
                 except UnboundLocalError or ValueError:
                     print("There is no file.")
+            case "3":
+                try:
```

Comment:
There is a syntax error.
You need to add `expect` clause as well when you use try clause.